extend-admin-ui-quick-start/README.md

```diff
@@ -0,0 +1,57 @@
+│ │ | ├── assets/
+│ │ | ├── css/
+│ │ | | ├── styles.css
+│ │ | ├── resources/
+│ │ ├── theme.properties
+
+In order to add a custom theme, in the `js/apps/keycloak-server/server/themes` folder, create a new folder named `mytheme` (can be any name you desire) which houses your personalized Admin UI theme. Within the `admin` sub-folder of this `mytheme` folder, there is a `theme.properties` file, along with a `resources` sub-folder that encompasses directories for CSS files, JavaScript files, and images. Within the `css` sub-folder, there is a `styles.css` file, which allows the addition of customized PatternFly styles. The `theme.properties` is a configuration file defines attributes such as the parent theme, and theme css. The `theme.properties` file is used to register the theme with the Keycloak server and integrate it with the Keycloak user interface.
+
+### Customizing with PatternFly Best Practices
+
+Overrides to PatternFly variables should be made at the `:root` level for global variables or at the top-level component selector for component variables (for example, `.pf-c-\*`), as these overrides will cascade down to children elements accordingly.
+
+To customize your Keycloak theme with PatternFly best practices, we recommend the following:
+
+1. To get started, you can utilize either our `extend-admin-ui-quick-start/src/mytheme` template or choose a theme from the theme directory located at `../lib/lib/main/org.keycloak.keycloak-themes-999.0.0-SNAPSHOT.jar`, which can be accessed using any standard ZIP archive tool.
+2. Use the provided PatternFly CSS classes when possible, rather than creating custom classes.
+3. Follow the recommended folder structure and naming conventions for your custom theme.
+4. Override the PatterFly variables at the `:root` level or at the top level component using the `pf-c-\*` naming convention (both examples are provided in the `styles.css` and [Patternfly documentation](https://www.patternfly.org/v4/developer-resources/global-css-variables/)).
+5. Create a custom stylesheet `(styles.css)` in the `resources/css` folder to contain your custom styles and overrides.
+6. Define your custom `styles=css/styles.css` in the `theme.properties` file. Make sure to add `parent=keycloak.v2` as parent theme for this custom theme. `title` of the Admin ui can be changed as well as `logo`.
+
+By following these best practices, you can ensure that your custom Keycloak theme remains consistent with the Keycloak Admin UI and is easy to maintain and update.
+
+### Applying a Customized Theme
+
+Start the Keycloak server by running the command `npm run start --workspace=keycloak-server`. To launch the Admin UI run the command `npm run dev`. After that, log in by accessing `http://localhost:8080/` and navigate to the `Themes` tab located in `Realm settings`. From there, change the Admin UI theme to your preferred custom theme. Finally, navigate to `http://localhost:8180/` to view the style you created being applied.
```

Comment:
Here I would also recommend just to tell to start the server and not this specific ui development commands. It's not necessary to run `npm run dev` and the server can be started with `./kc.sh` or `./kc.bat`